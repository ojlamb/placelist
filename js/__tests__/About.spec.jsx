import React from "react";
import { shallow } from "enzyme";
import About from "../components/about/AboutPage";

test("Search renders correctly", () => {
  const component = shallow(<About />);
  expect(component).toMatchSnapshot();
});
