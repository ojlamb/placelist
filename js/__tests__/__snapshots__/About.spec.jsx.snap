// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders correctly 1`] = `
<div
  className="AboutPage"
>
  <h1>
    About
  </h1>
  <p>
    Explain what the app does
  </p>
</div>
`;
